components/Search/SearchResultsForDocs.js

```diff
@@ -582,8 +582,9 @@ const SearchResultsForDocs = ({ apiResponse, entityType, context }) => {
             return (
               <FeedCard
                 {...post}
+                unifiedDocumentId={post.unified_document_id}
                 formattedDocType={"post"}
-                key={post?.id || index}
```

Comment:
Is this index fallback not necessary anymore?

Reply:
Yeah, it wasn't necessary to begin with